setup.py

```diff
@@ -1,107 +1,81 @@
 #!/usr/bin/env python3
-
-from setuptools import setup
-from setuptools.command.install import install as orig_install
-
-from distutils.core import Command
+import os.path
+import stat
+import sys
 from distutils.command.build import build as orig_build
-
-from os import path
-
-from sys import platform
-
-from urllib.request import urlopen
-from http import HTTPStatus
-
+from distutils.core import Command
 from hashlib import sha512
+from http import HTTPStatus
+from urllib.request import urlopen
 
-from os import makedirs
-from os import fstat
-from os import fchmod
+from setuptools import setup
+from setuptools.command.install import install as orig_install
 
-from functools import lru_cache
+SHELLCHECK_VERSION = '0.7.0'
+PY_VERSION = '1'
 
 
 def get_arch() -> str:
-    if platform == "linux" or platform == "linux2":
+    if sys.platform.startswith('linux'):
```

Comment:
this is what's suggested by the docs, however python3 always reports as `'linux'` so we could just use that